tests/framework/extensions/rke1/nodepools/nodepools.go

```diff
@@ -113,7 +113,7 @@ func updateNodePoolQuantity(client *rancher.Client, cluster *management.Cluster,
 			return false, err
 		}
 
-		if clusterResp.State == active && nodestat.AllManagementNodeReady(client, clusterResp.ID) == nil {
+		if clusterResp.State == active && nodestat.AllManagementNodeReady(client, clusterResp.ID, 30*time.Minute) == nil {
```

Comment:
Can the time duration be a const? It makes it easier to change in the future, if needed.